tests/integration/ie-tests.sh

```diff
@@ -14,57 +14,69 @@ test_example_job() {
 
   echo $K8S_MASTER_URL
 
+  PREVIOUS_JOB=$(kubectl --kubeconfig=${KUBE_CONFIG} get pods | grep driver | tail -n 1 | cut -d' ' -f1)
+
   # run the sample pi job using spark-submit
-  spark-client.spark-submit \
+  spark-client.submit \
+    --username=ie-test \
+    --log-level "DEBUG" \
     --master $K8S_MASTER_URL \
```

Comment:
shouldn't this be automatically supplied by the setup tool?

Reply:
you mean the master? yes, possibly.  @averma-canonical  Could we get rid of this?